src/tcwrapper.cpp

```diff
@@ -961,12 +961,12 @@ static void optimizemodule(TerraTarget *TT, llvm::Module *M) {
 
```

Comment:
Github won't let me comment on this, but right before this line is one that says `std::vector<llvm::Constant *> usedArray;` which I think should go away now.